Projects/UOContent/Commands/Possess.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using Server.Commands.Generic;
+using Server.Mobiles;
+
+namespace Server.Commands
+{
+    internal static class Swap
+    {
+        private static readonly HashSet<Layer> _specialLayer = new() { Layer.Backpack, Layer.Bank, Layer.Mount };
+
+        private static List<Item> GetItemsFromLayers(IEnumerable<Item> items) =>
+            new List<Item>(items).FindAll(item => !_specialLayer.Contains(item.Layer));
+
+        public static void Items(Mobile target, PlayerMobile caster)
+        {
+            var targetItems = GetItemsFromLayers(target.Items);
+            var casterItems = GetItemsFromLayers(caster.Items);
+
+            foreach (Item item in casterItems)
+            {
+                caster.RemoveItem(item);
+            }
+
+            foreach (Item item in targetItems)
+            {
+                target.RemoveItem(item);
+
+                if (!caster.EquipItem(item))
+                {
+                    Console.WriteLine($"Nie udało się ubrać {item.Name} na {target.Name}");
```

Comment:
I wish we had localization. Until we do, probably need to translate this from polish.
```cs
$"Cannot equip {item.Name} on {target.Name}"
```

Reply:
Also a few more questions:
Should this be a failure and force a revert? Maybe we can test for equipping before doing that so we don't get into an awkward state?

Next, should we `out string error` or something like that so we can do `AddResponse` instead of `Console.WriteLine`? 🤔

Reply:
This approach will indeed need to change to something else. Perhaps caster's clothes need to be packed in a container and stored away somewhere they can be retrieved from.

I was using the console log to demonstrate to my colleagues how it wouldn't work on a mob such as a chicken (ie. upon unpossessing you don't get your clothes back). ;)

Sorry, will be more mindful with commits on this branch going forward... or I can close the PR for now and re-open it once it's ready for review in a few days time.

Reply:
No problem, I am ok with messy commits. We squash when it gets merged in at the end. Thank you!